css/style.css

```diff
@@ -0,0 +1,12 @@
+body{
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    background: url(https://wallpaperaccess.com/full/2454628.png) no-repeat fixed center;
+}
+
+h1, p, form{
+    color: white;
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+}
+
```

Comment:
The CSS code it's very good :+1:, you only have to correct some indentation issues to make it look professional, please follow the linters suggests.